examples/vl3_basic/vl3-nse/cmd/vl3_connect.go

```diff
@@ -389,8 +410,8 @@ func removeDuplicates(elements []string) []string {
 	return result
 }
 
-// NewVppAgentComposite creates a new VPP Agent composite
-func newVL3ConnectComposite(configuration *common.NSConfiguration, ipamCidr string, backend config.UniversalCNFBackend, remoteIpList []string, getNseName fnGetNseName, defaultCdPrefix string) *vL3ConnectComposite {
+// newVL3ConnectComposite creates a new VL3 composite
+func newVL3ConnectComposite(configuration *common.NSConfiguration, ipamCidr string, backend config.UniversalCNFBackend, remoteIpList []string, getNseName fnGetNseName, defaultCdPrefix, ipamAddr, connDomain string) *vL3ConnectComposite {
```

Comment:
please change ipamAddr param name to something more like `svcRegistryServerAddr`.  `ipamAddr` is super confusing to what it actually is being used for.   Same comment for the vL3ConnectComposite struct.